package.json

```diff
@@ -296,6 +296,7 @@
     "ansi-regex": "5.0.1",
     "d3-color": "^3.1.0",
     "debug": "^4.3.4",
+    "fast-loops": "^1.1.4",
```

Comment:
I wonder how we're gonna deal with these resolutions later. A bunch of these pinned packages are here because there's no fixed version for the package we use at the time.

Since it's not easy to add comments in `package.json` I think we should periodically clean these pinned packages up.

Reply:
https://github.com/metabase/metabase/issues/45425